Packs/XMCO/Classifiers/classifier-mapper-incoming-XMCO.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "XMCO - Serenety Mapper",
+    "name": "XMCO - Serenety Mapper",
+    "type": "mapping-incoming",
+    "description": "Maps incoming XMCO alert fields.",
+    "fromVersion": "6.10.0",
+    "mapping": {
+        "XMCO Serenety Alert": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Rendered HTML": {
+                    "simple": "description"
+                },
+                "Sub Category": {
+                    "simple": "custom_fields.subcategory"
+                },
+                "XMCO Serenety identification": {
+                    "simple": "custom_fields.identification"
+                },
+                "XMCO Serenety monitoring": {
+                    "simple": "custom_fields.scope"
+                },
+                "rawJSON": {
```

Comment:
RawJson can't be mapped here and only through the integration py file. However, `Name` can be done here if you want. @jlevypaloalto - please assist with our best practice here.

Reply:
You are correct. The `rawJson` key is used in the python file only.

Reply:
Fixed :)